.papr.sh

```diff
@@ -6,10 +6,6 @@ if test -z "${INSIDE_CONTAINER:-}"; then
 
     if [ -f /run/ostree-booted ]; then
 
-        # by default, the root LV on AH is only 3G, but we need a
-        # bit more for our tests
-        lvresize -r -L +5G atomicos/root
```

Comment:
I think we still need this for `centos/7/atomic/smoketested` for now at least. I suspect this is waiting on inheriting the docker-storage-setup change to `overlay` by default. But I'll look into it.

Reply:
centos/7/atomic seems to be passing without this change.  Only fedora/27/atomic is failing at the moment (on `test_storage.sh`)